docs/developer/developer-setup.md

```diff
@@ -169,26 +169,34 @@ If you want to start developing on Trubudget, you need to setup the application
 
 This is the fastes way you can run all services needed for development. Everything is run in one command:
 
-1.  in root directory execute: `sh scripts/development/start-dev.sh`
+1.  in root directory execute: `sh scripts/development/start-dev.sh --full` or `sh scripts/development/start-dev.sh --slim`
 
-The script cares for `.env` file. No further step needed.
+This script sets the `.env` - file in the root directory automatically. No further step needed.
 
 Following services are dockerized:
 
 - Blockchain (master node)
 - API (master API)
 - Provisioning (feeds application with dummy data)
-- Excel export
 - Frontend
+- Excel export
+- Email notification
+- Email Database
+- Storage Service
+- Minio Database (external document storage)
 
```

Comment:
suggestion to add to the doc:
For further details how to use the script print out its help section with 
```
sh scripts/development/start-dev.sh -h
```

Reply:
done